### PR TITLE
Remove Rasmus email and add projects email

### LINKED ISSUE #179: Don't mention Rasmus
Make it clear that Dave V is in charge of Projects

### PR DESCRIPTION
**Pull request by davidventuri**
Fixes #179

### CODE REVIEW

File: docs/projects/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ If you've never seen a DataCamp project then you'll quickly get the gist by watc
 <center><iframe width="560" height="315" src="https://www.youtube.com/embed/HpVDKV2KALw" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe></center>
 
 > #### info::WANT TO CREATE A PROJECT? 
-> Currently only invited creators can create Projects on DataCamp, but we're always looking for engaged professionals and academics who want to help us teach Data Science. If you want to create a Project, please contact david.venturi@datacamp.com for more info!
+> Currently only invited creators can create Projects on DataCamp, but we're always looking for engaged professionals and academics who want to help us teach Data Science. If you want to create a Project, please email projects@datacamp.com for more info!
```
Comment (richierocks) [inline docs/projects/README.md]: Might be worth consistently making these links clickable, as you have elsewhere. `[projects@datacamp.com](mailto:projects@datacamp.com)`

File: docs/projects/projects-process.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 The Project creation process
 =============================================
 
-This page details steps you go through when you create a *premium* DataCamp Project. If you're interested in creating a premium Project, please contact david.venturi@datacamp.com for more info!
+This page details steps you go through when you create a *premium* DataCamp Project. If you're interested in creating a premium Project, please email projects@datacamp.com for more info!
```
Comment (richierocks) [inline docs/projects/projects-process.md]: Same.

File: docs/projects/projects-process.md [hunk 3]
```diff
@@ -69,7 +69,7 @@ To this end, here are some guidelines for what makes a good Project:
 
 It is generally no problem if you want to re-purpose already existing material (such as a blog post or existing case study) as a DataCamp project.
 
-Fill out the following information and email it to david.venturi@datacamp.com:
+Fill out the following information and email it to projects@datacamp.com:
```
Comment (richierocks) [inline docs/projects/projects-process.md]: Same.

**Comment by davidventuri** [comment]
@richierocks Yes, good idea. This should be fixed now.

**Pull request merged**